gatsby-config.js

```diff
@@ -1,40 +1,41 @@
-const path = require('path');
-const { getCrumbLabelUpdates } = require('./scripts/breadcrumbs');
+const path = require("path");
+const { getCrumbLabelUpdates } = require("./scripts/breadcrumbs");
 
 require("dotenv").config({
   path: `.env.${process.env.NODE_ENV}`,
-})
+});
 
 // Get paths of Gatsby's required rules, which as of writing is located at:
 // https://github.com/gatsbyjs/gatsby/tree/fbfe3f63dec23d279a27b54b4057dd611dce74bb/packages/
 // gatsby/src/utils/eslint-rules
 const gatsbyRequiredRules = path.join(
   process.cwd(),
-  'node_modules',
-  'gatsby',
-  'dist',
-  'utils',
-  'eslint-rules'
+  "node_modules",
+  "gatsby",
+  "dist",
+  "utils",
+  "eslint-rules",
 );
 
 module.exports = {
   trailingSlash: "always",
   siteMetadata: {
     siteUrl: `https://designers.italia.it`,
-    title : "Designers Italia",
-    author : "Dipartimento per la trasformazione digitale e AGID",
-    lang : "it",
-    description : "Il punto di riferimento per la progettazione dei servizi pubblici digitali",
-    siteName : "Designers Italia",
-    image : "https://designers.italia.it/assets/icons/logo-it.png",
-    twitterImage : "https://designers.italia.it/assets/icons/logo-it.png",
-    twitterSite : "@DesignersITA",
-    twitterCreator : "Dipartimento per la trasformazione digitale e AGID",
-    themeColor: "#0066cc"
+    title: "Designers Italia",
+    author: "Dipartimento per la trasformazione digitale e AGID",
+    lang: "it",
+    description:
+      "Il punto di riferimento per la progettazione dei servizi pubblici digitali",
```

Comment:
@bfabio this could be on row 27 after `:`?

Reply:
@Fupete This is `prettier` acting on https://prettier.io/docs/en/options.html#print-width, that's expected and good